src/libpmemfile/path_resolve.c

```diff
@@ -69,7 +69,7 @@ get_stat(struct resolved_path *result, struct stat *buf)
 		int r = pmemfile_fstatat(result->at.pmem_fda.pool->pool,
 			result->at.pmem_fda.file,
 			result->path,
-			buf,
+			(struct pmemfile_stat *)buf,
```

Comment:
Then get_stat user will have to use pmemfile_stat, which means more code will have to use it and I think it doesn't make much sense.